iib/web/static/api_v1.yaml

```diff
@@ -460,8 +461,8 @@ paths:
                 properties:
                   error:
                     type: string
-                    example: '"organization" must be a string. This occurred on the build request '
```

Comment:
Consider:
```yaml
example: >-
  "organization" must be a string. This occurred on the build request
  in index 0.
```

Reply:
Merged https://github.com/release-engineering/iib/pull/128.
Could you please rebase this PR on that?

Reply:
OK, rebased on top of master.